packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add a new `DeFiPositionsController` that keeps an update list of DeFi positions for EVM accounts ([#5400](https://github.com/MetaMask/core/pull/5400))
+  - Export `DeFiPositionsController`
+  - Export the following types
+    - `DeFiPositionsControllerState`
+    - `DeFiPositionsControllerAllowedActions`
+    - `DeFiPositionsControllerAllowedEvents`
+    - `DeFiPositionsControllerActions`
+    - `DeFiPositionsControllerEvents`
+    - `DeFiPositionsControllerGetStateAction`
+    - `DeFiPositionsControllerStateChangeEvent`
+    - `DeFiPositionsControllerMessenger`
+    - `GroupedDeFiPositions`
+  - Add `@metamask/transaction-controller@^45.0.0` as a peer and dev dependency
```

Comment:
Nit can we mark this as BREAKING

Reply:
Is adding a new dependency a breaking change though?

Reply:
Yes, adding or bumping a peer dependency is considered a breaking change, since it requires consumers to install and use the same version of that dependency to avoid compatibility issues.